templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+<div class="lune-page lune-page-product {{classes}}">
+  <div class="md-col-9 mx-auto {{classes}}">
```

Comment:
repeating classes across 2 divs ? in future if we add classes it might break , i would recommend to have just one wrapping div.